.github/workflows/scan-for-cves.yaml

```
@@ -41,3 +41,12 @@
         with:
           sarif_file: reports
           # TODO: github dependency tree?
+
+  createIssue:
+    runs-on: ubuntu-latest
+    name: Create Issue
+    if: steps.generateSarifReports.conclusion == 'failure'
+    steps:
+      - uses: JasonEtco/create-an-issue@v2
+        env:
+          GITHUB_TOKEN: ${{GITHUB_TOKEN}}
```

actionlint (1.7.7)

undefined variable "GITHUB_TOKEN". available variables are "env", "github", "inputs", "job", "matrix", "needs", "runner", "secrets", "steps", "strategy", "vars"

(expression)